src/common/domain/barrel/data/BarrelDetailsStore.ts

```diff
@@ -4,6 +4,7 @@ import BarrelDetailsResponse from "../model/BarrelDetailsResponse";
 import BarrelType from "../BarrelType";
 import { getComponentsFromSections } from "../../../../coco/zeplinComponent/util/zeplinComponentUtil";
 import BaseError from "../../error/BaseError";
+import { getProjectJiras, getProjectItemJiras } from "../../jira/util/jiraUtil";
```

Comment:
`"../../jira/util/jiraUtil"` 

Isn't this verbose?

Reply:
see this comment
https://github.com/zeplin/vscode-extension/pull/26#discussion_r411256941